yw_website/yw_website/apps/website/views.py

```diff
@@ -90,17 +90,34 @@ def my_workflows(request):
 
 def edit_workflow(request, workflow_id, version_id):
     workflow = Workflow.objects.get(pk=workflow_id)
+    workflowTags = TagWorkflow.objects.filter(workflow=workflow_id)
     if request.method == "POST":
         form = request.POST
         title = request.POST.get("title")
-        tags = request.POST.get("tag_arr")
+        tags = request.POST.getlist("tagArray")
+        t = ''.join(tags)
+        tStr = str(t)
+        tagArr = tStr.split(',')
+        if tags:
+            newTag = tagArr
```

Comment:
should use snake case instead of camel case so these variables go well with the rest of the project convention